### ISSUE TITLE
Exports by ordinal shall not have hint

### ISSUE THREAD

**Issue by MarekKnapek** [lucasg/Dependencies#123]
If some DLL exports a function by ordinal only (not by string name) it shall not have a hint. Hint is for name exports. It is a hint index into name pointer table to kick off binary search for that name. I'm not sure why would you subtract one from the ordinal and tread it as hint. Note that ordinal base is not always equal to one. Please see attached screenshots. Note that you have similar issue when all exports are ordinals and none of them is name, this issue manifests itself even if there is some exports by ordinal and some exports by name.

![issue109a](https://user-images.githubusercontent.com/13893067/73657154-fc944880-4691-11ea-9698-1c1ef2a1c62f.png)

![issue109b](https://user-images.githubusercontent.com/13893067/73657167-0027cf80-4692-11ea-9a67-44815a48fbaa.png)

![issue109c](https://user-images.githubusercontent.com/13893067/73657172-01f19300-4692-11ea-9364-c742bd6c6900.png)

![issue109d](https://user-images.githubusercontent.com/13893067/73657177-04ec8380-4692-11ea-880f-a3470582024b.png)

**Comment by MarekKnapek** [comment]
Oh I forgot: I tested this on v1.9.312 downloaded from https://ci.appveyor.com/project/lucasg/dependencies/builds/30521285.

**Comment by lucasg** [comment]
Okay fixed it : 994ba58790f88e381f9fc0ac2c8d635106225f01
(wtf, now github automagically fix issues based on commit title ?)

**Comment by MarekKnapek** [comment]
Hi Lucas, I think you meant commit 1d6f1a07e3123fc19150a4f95bef98f80844247f but you mentioned commit 994ba58790f88e381f9fc0ac2c8d635106225f01 instead.

I tested build version 1.9.321 downloaded from https://ci.appveyor.com/project/lucasg/dependencies/builds/31478826 and I confirm that there is no longer an invalid hint for ordinal-only exports. Good job. Thank you.

Unfortunately, in the original comment I mentioned that I don't understand that you how you compute the hint. You just subtract one from ordinal and declare it as hint. This is wrong, I created issue #130 for you to fix this bug.
